src/constraints/create.jl

```diff
@@ -202,10 +213,46 @@ function _create_constraints_tables(connection)
             ON t_high.asset = asset.asset
         WHERE
             asset.type in ('producer', 'conversion')
-            AND asset.unit_commitment = true;
+            AND asset.unit_commitment
+        ",
+    )
+
+    DuckDB.query(
+        connection,
+        "CREATE OR REPLACE TEMP SEQUENCE id START 1;
+        CREATE OR REPLACE TABLE cons_max_output_flow_with_basic_unit_commitment AS
+        SELECT
+            nextval('id') AS index,
+            t_high.*
+        FROM t_highest_assets_and_out_flows AS t_high
+        LEFT JOIN asset
+            ON t_high.asset = asset.asset
+        WHERE
+            asset.type in ('producer', 'conversion')
+            AND asset.unit_commitment
+            AND asset.unit_commitment_method = 'basic'
+        ",
+    )
+
+    DuckDB.query(
+        connection,
+        "CREATE OR REPLACE TEMP SEQUENCE id START 1;
+        CREATE OR REPLACE TABLE cons_max_ramp_with_unit_commitment AS
+        SELECT
+           nextval('id') AS index,
+           t_high.*
+        FROM t_highest_assets_and_out_flows AS t_high
+        LEFT JOIN asset
+            ON t_high.asset = asset.asset
+        WHERE
+            asset.type in ('producer', 'conversion')
+            AND asset.ramping
+            AND asset.unit_commitment
+            AND asset.unit_commitment_method = 'basic'
         ",
     )
 
+    # TODO: Should there be a `AND NOT asset.unit_commitment` in the WHERE here?
```

Comment:
@datejada, this was a question I had when looking at these tables. The `without_unit_commitment` doesn't check for `asset.unit_commitment = false` or `NOT asset.unit_commitment`

Reply:
Good catch! We need to add those checks in the SQL query. Thanks!